aws/resource_aws_waf_byte_match_set.go

```diff
@@ -107,7 +107,7 @@ func resourceAwsWafByteMatchSetRead(d *schema.ResourceData, meta interface{}) er
 	}
 
 	d.Set("name", resp.ByteMatchSet.Name)
-	d.Set("byte_match_tuples", flattenWafByteMatchTuples(resp.ByteMatchSet.ByteMatchTuples))
```

Comment:
When renaming attributes, we need to ensure both the old and new are still read into the Terraform state.